### ISSUE TITLE
Minor comments

### ISSUE THREAD

**Issue by ericagol** [ericagol/TRAPPIST1_Spitzer#20]
1. Introduction
- [x] Please, check the equation (1) and give a citation.  @danfabrycky ?
- [x] How long is t_survey exactly?
- [x] Please, also cite Delrez et al. (2018) in the sentence "...larger uncertainties in the masses of the planets (Gillon et al. 2017; Grimm et al. 2018)".

2. New Trappist-1 observations
- [x] The authors describe that they used 284 previous, and additional 163 transits from a combination of space- and ground-based observations. Please, mention here that 447 transits were used, but 61 transits were duplicates, so in fact only 386 individual transit events were used during the TTV analysis.
- [x] It would be very useful to mention here, which Spitzer IRAC channels were used. How many transits were observed in each channel?
- [x] "Differential photometry was then used after a careful selection of comparison stars...". What does mean careful? What was the criterion for selection? Which stars were selected finally? Please, describe this in more details. @educrot 
- [x] Provide exact coordinates for the comparison stars used 

- [x] If K2 observed the TRAPPIST-1 system during campaingns 12 and 19, why only campaign 12 data were used? Did the authors apply dilution correction during K2 data reduction?
- [x] "For each planet a fixed time of transit for epoch zero (T_0) and fixed period (P) were used..." Give exact values, how were P and T_0 fixed.  @educrot 

3. Excess of outliers and robust likelihood model:
- [x] Please, mark in Fig. 1 the mentioned 8 data points with z<-3 and 7 points with z>3 for better clarity.
- [x] Can the authors present a light-curve example, which shows the described discrepancy?
- [x] What could be the explanation that Grimm et al. (2018) did not report about such outliers?
- [x] Typo: Levenburg ==> Levenberg. Please, check this typo in the entire draft.

4. Transit-timing analysis
- [x] Please, clarify, when you are describing observed and calculated transit times. Now, it is not clear from the text.
- [x] During the HMC analysis the authors used 37 free parameters. Please, repeat here, which parameters were used, and why are 37 parameters in the model: because (5*7)+2.
- [x] Reorder Figs. 4, 5, and 6. Now, as first is mentioned Fig. 6, then 5, and finally Fig. 4.
- [x] "1 - sigma" is very confusing, it should be simple "1sigma". Also, it should be described that the final parameter value corresponds to median value, and that 1sigma corresponds e.g. to +/- 34.1%.
- [x] Fig. 2 contains very strange color combinations, it is hard to see green on blue, mainly in the cases of T1d and T1d. Please, choose another color combination.
- [x] In Fig. 4, mark +/- 1sigma intervals e.g., with vertical lines for better comparison with the results presented in Grimm et al. (2018). Also, please, mark the midpoint of intervals adopted from Grimm et al. (2018) differently, e.g., with circles.
- [x] In Table 2, the first column should be simple "mass of the planet in Earth masses" M_p [M_Earth], if I understand this correctly. 
- [x] In Table 3 similarly. I also recommend to prepare an Rp comparison using the values presented in Delrez et al. (2018).
5. Photodynamical analysis
- [x] Since the fitted parameter is Rp/Rs, in Table 4 should be also Rp/Rs [-] and not Rp [Rs].
- [x] Please, reorder these sentences: "The posterior distribution of the result of the fit are given in Table 5. We used a prior which places bounds on each parameter given in Table 4".
- [x] Can the authors show period-folded light curves corrected for the measured TTVs?
- [x] Regarding to equation (14), can the authors show such an equation, where the eccentricity is not neglected and describe, which part of the equation was neglected due to the small eccentricities? 
- [x] "We conclude that it is likely that all of the planets orbit on the same side of the star..." This should not be a conclusion, this is a degeneracy of the definition of inclination (0-90 deg). We could not distinguish e.g., between 85 and -85 deg, therefore it should be stated that planetary positions on the star chord is arbitrary. According to this, the statement about the planetary orbit flatness should be more conservative. It would be also useful a plot, showing a stellar disk and the planetary orbits in the observer's view, with x and y axes in units of impact parameter.

6. Planet densities and mass-radius relation
- [x] Please, correct the following: "M_star = (0.0898 +/- 0.0023) M_odot" should be without parentheses. Also, similarly in other cases.
- [x] In caption of the Table 5 it should be corrected the "radius-ratio of planet to star" ==> "planet-to-star radius ratio". I recommend to correct and unify this in the entire text.
- [x] Ingress duration "tau" is presented only for ingress (egress) time, or ingress + egress time? Please, clarify this.
- [x] Unit g/cc should be g/cm^3, if I understand this correctly.
- [x] In Fig. 12 adjust the equilibrium temperature scale to the color borders.

7. Stallar parameters
- [x] Please, describe, how did you calculate Teff, presented in Table 7. Furthermore, it is not clear to me, how did the authors get the uncertainties, presented in Table 7. Please, clarify this, as well.

8. Search for an eight planet
- [x] It is not necessary to describe that "This analysis was carried out on four Hyak nodes..." See my major comment.

9. Interior compositions
- [x] The authors conclude that all seven planets appear to be consistent with a line of interior composition and that the composition may vary little from planet to planet. I would like to recommend to say this more conservatively, according to the 2sigma uncertainties, presented in Fig. 12. Mainly this is true for the planets c, b, e, and h.
- [x] Since the caption of Fig. 15 contains relevant information, it should be included to the main body of the text, except the first sentence.
- [x] Please, crop Fig. 16 to the real information and eliminate unnecessary white areas in this graph.
- [x] Abbreviation "CMF" is not explained in the text.
10. Discussion
- [x] Please, cite the appropriate papers ... when discussing stellar activity.
 http://dx.doi.org/10.1088/0004-637X/791/1/55 
http://dx.doi.org/10.1051/0004-6361/201424059
**(Also, Rackham et al., which is already cited)**
- [x] It is not clear to me, how the authors got eccentricities, presented in Fig 21, from e*sin omega and e*cos omega.
**e = sqrt((e*cos(omega))^2 + (e*sin(omega))^2)**
- [x] "The stellar density which was derived using the photodynamic model, ..., is moderately higher and more precise..." I recommend to correct the "moderately higher" ==> "in 1sigma agreement with prior analysis".

11. Conclusions
- [x] Regarding to conclusions about uniform planetary compositions and uniform inclinations, please check my minor comments described above.

**Comment by educrot** [comment]
Concerning the following comment:

>  "Differential photometry was then used after a careful selection of comparison stars...". What does mean careful? What was the criterion for selection? Which stars were selected finally? Please, describe this in more details.

With "careful" I meant comparison stars in TRAPPIST-1's  vicinity with magnitude and spectral type as close to the ones of TRAPPIST-1 as possible.  We can maybe drop the word "careful" though. 
To do:

- [ ] Provide exact coordinates of the comparison stars used

**Comment by danfabrycky** [comment]
Hi Eric,

Regarding #20, I wanted to know if the equation works when applied to b/c
pair and c/d pair, near 8:5 and 5:3 respectively, with k=3 and 2 in the
numerator, respectively. So I ran simulations of the pairs (M*=0.09 Msol)
-- 

bc_sim:

planet    period (d)     T0 (d)            e           i (deg)   Omega (deg)
  omega (deg)   mp (mjup)

01=b 1.510826 7257.55044 0.0034 90           0.0 135.0 0.00428

02=c  2.421937 7258.58728 0.00055 90     0.0 1.0 0.00408


and

cd_sim:

planet    period (d)     T0 (d)            e           i (deg)   Omega (deg)
  omega (deg)   mp (mjup)

01=c  2.421937 7258.58728 0.00055 90     0.0 1.0 0.00408

02=d 4.049219 7257.06768 0.00600 90      0.0 150.0  0.00121


Then I fourier-decomposed the resulting O-C curves, which spanned the time
in our figure 2. See attached.

What I found was that no frequency dominates, and the super-period existing
elsewhere in the system, the 487 day super-period aka 0.00205 day^{-1}
super-frequency, simply isn't showing up.

The frequencies that do show up can be identified with the offset from ...
[for bc pair]  3:2 resonance (0.0851day^{-1}), 2:1 resonance
(0.1639day^{-1}), 1:1 resonance (0.2490day^{-1})
[for cd pair]  5:3 resonance (0.00445day^{-1}, i.e. 224.2days),
2:1 resonance (0.0808day^{-1}), 3:2 resonance (
0.0853day^{-1}), 1:1 resonance (0.1661day^{-1})

On the longer-period planet of each pair, the 1:1 resonance folds back on
top of the 2:1 location due to Nyquist sampling -- for c in the bc pair,
the 1:1 resonance appears at 0.1639day^{-1}; for d in the cd pair, the 1:1
resonance appears at 0.0808day^{-1}.

I think the 5:3 resonance shows up for the cd pair whereas the 8:5 didn't
show up for the b/c pair because both eccentricities for b and c are very
small, but in the c/d pair the eccentricity of d is sizable. Now in the
real system, planet d's eccentricity is precessing once every 487 days, so
that may actually matter for that component, and I suppose it could shift
it to 487 days in synchrony with the rest of it. So this suggests that the
multi-resonant chain's TTV cannot be simplified to pairwise interaction
(unlike Kepler-11, where most of the planets are dominated by non-resonant
chopping interactions, which can be so decomposed) -- not a huge surprise!

I'm happy to discuss more what should be done with equation 1. To me the
above suggests throwing a k in the numerator doesn't do it justice.

Dan

**Comment by ericagol** [comment]
Regarding the comment above of providing an example of a discrepant transit, here is a Spitzer transit of planet b which occurs later by 3.3sigma.

![Example_outlier_Sp_b_7827 15](https://user-images.githubusercontent.com/243664/92666235-8c037900-f2bd-11ea-8671-e57b6e6b89ab.png)

**Comment by danfabrycky** [comment]
Now, looking back at my email, I should have explicitly reported:
The TTV period associated with 5:3 was 1/(5/P_e - 3/P_d), i.e. no 2 in the
top.
To replicate the TTV frequencies I reported, one needed to look at the
figures and use periods of:
1.51083d and 2.42186d in the b/c simulation, and
2.42154d and 4.05048d in the c/d simulation,
rather than the oscillating periods I started the simulations with.

I also just now did a higher eccentricity (0.01 for each) version of the
b/c simulation, and a frequency pops up now at
bc: 8:5 (0.0067 day^{-1}, i.e., 149days)
So that is just 1/(8/P_b - 5/P_c) with no 3 on top.

Dan
